### DISCUSSION TITLE
I experience mixed outputs in Chinese, Korean, and other languages in Camel Role-Playing

### DISCUSSION THREAD

**Discussion by coolbeevip** [camel-ai/camel#978] [General]
I experience mixed outputs in Chinese, Korean, and other languages while using Camel Role-Playing. 

I'm not sure if this is caused by gpt-4o-mini.

This issue can be reproduced through the script at https://gist.github.com/coolbeevip/82f914500bd9b5b59c2ce99688153341.

**Comment by Wendong-Fan** [comment]
@coolbeevip  thanks for open this discussion topic! From the output it's really weird so I did some search to see whether there's anyone else met the same issue, I found one article here: https://futurism.com/the-byte/chatgpt-problem-chinese
I guess it's due to some Chinese data was not well-cleaned when doing model training for the model..
